draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -105,30 +105,30 @@ Selectively Disclosable JWT (SD-JWT)
 :  A JWT [@!RFC7515] created by the issuer, which is signed as a JWS [@!RFC7515],
    that supports selective disclosure as defined in this document.
 
-SD-JWT Salt/Value Container (SVC)
-:  A JSON object created by the issuer that contains mapping between
+Issuer-Issued Disclosures Object
```

Comment:
Shall we explain what a "Disclosure" is before this?

"Information suitable to disclose the cleartext claim value and potentially other information, such as a salt and a claim name, to a Verifier."

Reply:
I like the idea!

Reply:
paraphrased to 

Disclosure
: A combination of the cleartext claim value, the cleartext claim name, a salt and
   optionally blinded claim name value that is used to calculate a digest for a certain claim.